lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -100,24 +100,24 @@ def inversepower(x, base=2):
     except TypeError, e:
         raise errors.AnsibleFilterError('root() can only be used on numbers: %s' % str(e))
 
+size_ranges = (
+    (1<<80L, 'Y'),
+    (1<<70L, 'Z'),
+    (1<<60L, 'E'),
+    (1<<50L, 'P'),
+    (1<<40L, 'T'),
+    (1<<30L, 'G'),
+    (1<<20L, 'M'),
+    (1<<10L, 'K'),
+    (1, '')
+)
 
 def human_readable(size, isbits=False, unit=None):
 
     base = 'bits' if isbits else 'Bytes'
     suffix = ''
 
-    ranges = (
-            (1<<70L, 'Z'),
-            (1<<60L, 'E'),
-            (1<<50L, 'P'),
-            (1<<40L, 'T'),
-            (1<<30L, 'G'),
-            (1<<20L, 'M'),
-            (1<<10L, 'K'),
-            (1, base)
```

Comment:
I'll take a look. If this is the case, I'll add a new test in test/integration playbook.